app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -246,6 +246,7 @@ class RawSubmissionTable extends React.Component {
           ref={(r) => this.actionBox = r}
           disabled={this.props.selection.length === 0}
           is_admin={this.props.is_admin}
+          is_ta={this.props.is_ta}
```

Comment:
Please use 'grader' instead of 'ta' wherever possible. I know that we use ta in a bunch of places but I'm trying to get rid of that language where we can.